Dockerfile.linux_amd64

```diff
@@ -0,0 +1,13 @@
+FROM amd64/ubuntu
+RUN apt-get update && \
+    apt-get install -y protobuf-compiler git cmake gcc g++
+WORKDIR /workdir
+COPY . /workdir
```

Comment:
We should map local directory to container instead of copying source to the image.